app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -250,6 +251,16 @@ val networkModule = module {
         objectMapper
     }
 
+    single {
+        PagedList
+            .Config
+            .Builder()
+            .setPageSize(5)
```

Comment:
so we are requesting for more data after 5 elements right? I guess this is too less. We can load 15-20 events at time if it doesn't take too much time to load.

Reply:
I think it is fine. Much better for mobile data. Will test locally

Reply:
@iamareebjamal did you test? This PR looks good otherwise, let's finalise it